libs/seiscomp/seismology/ttt/descriptions/global_homogeneous.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<seiscomp>
+	<plugin name="homogeneous">
+		<extends>global</extends>
+		<description>Travel time table for a homogeneous velocity model</description>
+		<configuration>
+			<group name="ttt">
+			  <group name="homogeneous">
+				<description>
+				 Travel-time interface for homogeneous velocity models.
+				</description>
+					<group name="profile">
+						<description>
+							Parameters defining the area where the profile applies and the P/S velocities.
+							Once defined, the profile must be registered in ttt.homogeneous.tables
+						</description>
+						<struct type="homogeneous profile" link = "ttt.homogeneous.tables">
```

Comment:
Actually the "link" is not present in this description. As far as I see the code is not relying on traversing available tables so the registration is not necessary.

Reply:
I'll move the "link" in the description of group "profile", where the link is actually used. I'll also change "must be registered" with "can then be registered"